rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/activity/OneScanActivity.java

```diff
@@ -91,6 +92,7 @@ private void setResultAndFinish(OneScanHelper.ScanResponse response) {
                 response.lot, response.productId, String.valueOf(response.sensorTriggered),
                 response.status}, ',');
         resultIntent.putExtra(BARCODE_KEY, barcode);
+        resultIntent.putExtra(CovidConstants.IntentKeys.BARCODE_SENSOR_TRIGGER, response.sensorTriggered);
```

Comment:
Why is this needed? I think this value can be extracted from the barcode?

Reply:
fixed.